src/tools/sdktool/addmersdkoperation.cpp

```diff
@@ -326,6 +326,7 @@ int AddMerSdkOperation::execute() const
     if (map.isEmpty())
         map = initializeSdks(m_version);
 
+    map.insert(QLatin1String(Mer::Constants::MER_SDK_FILE_VERSION_KEY), version);
```

Comment:
version seems to be undefined here

Reply:
hm, it seems I tried building wrong subdir... fixed, thanks!